api/src/controllers/client/options.ts

```diff
@@ -94,5 +88,13 @@ export async function clientOptions(
   const output = await Promise.all(req.body.map((v) => findOptions(v, user)))
   const toc = process.hrtime(tic).reduce((sec, nano) => sec * 1e3 + nano * 1e-6)
   logger.info('%s: returning options in %d ms', user.username, toc.toFixed(0))
-  return res.status(output.some((v) => v.error) ? 404 : 200).json(output)
+  return res
+    .status(
+      output.some((v) => v?.error === 'batch sealed')
+        ? 409
+        : output.some((v) => v?.error === 'team not found')
+        ? 404
+        : 200
+    )
+    .json(output)
```

Comment:
Unrelated bugfix: We should always return proper status code if we encountered an error.